src/cs3panel.tsx

```diff
@@ -89,18 +93,67 @@ export class Cs3HeaderWidget extends ReactWidget {
     }
 }
 
+export const Bottom = (props: BottomProps): JSX.Element => {
+
+    const [text, setText] = useState(props.message);
+
+    const onPathOrFileChanged = async (path: string) => {
+        const showHidden: boolean = await props.db.fetch('showHidden') as boolean;
+        if (showHidden == undefined || !showHidden) {
+            const result: Contents.IModel = await requestAPI('/api/contents/' + path, {method: 'get'});
+            const hiddenFilesNo = result.content.filter((file: { name: string; }) => file.name.startsWith('.')).length
+            setText('Hidden files ' + hiddenFilesNo + ' (show)')
+        } else {
+            setText('Cover hidden files')
+        }
+    }
+
+    props.browser.model.pathChanged.connect(async (browser, args) => {
```

Comment:
Not knowledgeable enough on lab inner workings, but don't we have a way to subscribe to contents changed and fetch the number information from there?

Reply:
Unfortunately browser.model events don't expose info about current content, or the access is restricted